wiki/Installing-Oppia-Android.md

```diff
@@ -366,61 +366,31 @@ Please follow these steps to set up Oppia Android on your local machine.
 
-6. A file similar to the one below will be generated:
+Click on the 3 dots to open the file picker. Scroll down and select `oppia-android.bazelproject`, then click **next**.
 
-   <img width="760" alt="Image" src="https://github.com/user-attachments/assets/5a26210f-865d-4048-b5a5-73379984bfb5" />
+   <img width="991" alt="Screenshot 2025-03-19 at 18 45 23" src="https://github.com/user-attachments/assets/31e0694c-a179-4adf-8d4d-28793c586ee5" />
 
-7. In the above file, modify it so that it looks like below (copy and paste):
-   
-```
-directories:
-  # Add the directories you want added as source here
-  # By default, we've added your entire workspace ('.')
-  .
-  
-# Automatically includes all relevant targets under the 'directories' above
-derive_targets_from_directories: false
-
-targets:
-# If source code isn't resolving, add additional targets that compile it here
-  //:oppia_dev_binary
-  //app/src/main/... 
-  //domain/src/main/... 
-  //testing/src/main/... 
-  //utility/src/main/... 
-  //data/src/main/... 
-  //model/...
-  
-additional_languages:
-  # Uncomment any additional languages you want supported
-  # c
-  # dart
-  # kotlin
-  java
-  kotlin
+6. The project view will be generated as follows. Click the **Create** button to complete the import.
 
-# Please uncomment an android-SDK platform. Available SDKs are:
-# android_sdk_platform: android-28
-# android_sdk_platform: android-29
-android_sdk_platform: android-30
-# android_sdk_platform: android-31
-# android_sdk_platform: android-32
-# android_sdk_platform: android-33
-android_sdk_platform: android-34
+   ![Screenshot 2025-03-19 at 18 47 57](https://github.com/user-attachments/assets/b7016ab8-9ae5-44e0-a624-c7cc8fc576da)
 
-shard_sync: true
+7. Immediately, the project will begin to synchronize, and will be ready once sync completes. Read more about syncing in the [Bazel User Guide](https://github.com/oppia/oppia-android/wiki/Bazel-User-Guide#Syncing-the-project).
 
-java_language_level: 11
-```
-   **Note**: Only enable Android-SDK platforms that you have installed per the [Prepare developer environment](#prepare-developer-environment) section above. Android-SDK platforms are useful for creating emulators to test your app with.
+   ![Screenshot 2025-03-19 at 18 49 59](https://github.com/user-attachments/assets/603d85d6-1336-4893-8153-eef8caef5e81)
 
-8. Click **Create**, and allow the project to synchronize. Read more about syncing at [Bazel User Guide](https://github.com/oppia/oppia-android/wiki/Bazel-User-Guide#Syncing-the-project).
+**Note**: Unlike Gradle, Bazel does not support the "**Android**" project view. To see all project directories, switch to the "**Project**" view instead.
+
+In the `.aswb` directory, you will find the generated `.bazelproject` file. It should contain:
+
+```shell
+import oppia-android.bazelproject
+```
```

Comment:
Ooh, so importing a project view actually just generates the 'import' statement on its own? I want to double check this before approving since I expected that it would copy it.

Reply:
I tested the import, and this was the outcome.